.golangci.yml

```diff
@@ -0,0 +1,26 @@
+linters:
+  auto-fix: false
+  enable:
+    - nakedret 
+    - interfacer
+    - varcheck
+    - deadcode 
+    - structcheck 
+    - misspell
+    - maligned 
+    - ineffassign
+    - goconst
+    - goimports
+    - errcheck 
+    - dupl 
+    - unparam 
+    - golint 
+    - staticcheck
+    - unused
+    - gosimple
+    # missing to be added
+    # - lll (250), gocyclo, gosec, scopelint, unconvert, stylecheck
+run:
+linters-settings:
+  errcheck:
+    check-type-assertions: true
```

Comment:
Why this check is required?

Reply:
this is the "extra" check that I usually run with that was capturing the lack of error handling in `olm.go`.   Removing this, removes the need for change.